doc/STORY.txt

```diff
@@ -54,85 +54,85 @@
  Three heroines of Gensōkyō sat drinking tea on a calm day at the end of summer.
  Two of them seemed unsettled, while the third—
 
- Marisa: “What’s got ya both so down? Did I mess up the tea?”
+ Marisa: ”What’s got ya both so down? Did I mess up the tea?”
 
  Reimu and Yōmu shared a knowing glance.
 
- Reimu: “The tea’s fine. I just feel weird. I can’t put my finger on it.”
+ Reimu: ”The tea’s fine. I just feel weird. I can’t put my finger on it.”
 
- Yōmu: “Likewise.“
+ Yōmu: ”Likewise.”
 
  The bells of the shrine tolled outside. Someone had made a donation. A minute
  later, there was another. And another.
 
- Reimu: “Something’s definitely not right…”
+ Reimu: ”Something’s definitely not right…”
 
- Yōmu: “Isn’t it normally comforting for the shrine to receive donations?”
+ Yōmu: ”Isn’t it normally comforting for the shrine to receive donations?”
 
- Reimu: “Normally it *would* be comforting, but…“
+ Reimu: ”Normally it *would* be comforting, but…”
```

Comment:
Sorry, I hate to be pedantic about it, but this change actually broke most of the quotes in this file. They were *mostly* right before, except for a few odd cases I missed during the last review. But the change made those broken in the opposite way.

Example:
![screenshot](https://0x0.st/iX8k.png)

The top two lines used to have correct quotes. The bottom line used to have two opening quotes, now it has two closing quotes.

On the other hand, the endings file seems clean now, thanks.

Reply:
That's what I get for pushing stuff at 2AM. Should be fixed now.